src/Conf.ml

```diff
@@ -369,6 +369,13 @@ module Formatting = struct
         , "$(b,before) breaks the expressions before the separator." ) ]
     in
     C.choice ~names ~all ~doc ~section
+      ~removed_values:
```

Comment:
@Julow is this suggestion accurate?

Reply:
You also need `space-around-lists=false`, otherwise there will be a break before the closing `]`.

Reply:
done!